CHANGELOG.md

```diff
@@ -1,216 +1,51 @@
+| :---- | :----------- | :----------------------- | :------------------------------------------------------------------------------------ | :----------- | :-------------- | :--------- |
+| Task  | Done         |                          | Publish new Magento PWA meta package https://github.com/magento-commerce/magento2-pwa | [PWA-2329][] | No PR           | PWA-12.1.0 |
+| Bug   | Deployment Q | Tommy Wiebell            | Cypress snapshots outdated with newsletter in footer                                  | [PWA-2287][] | [3506][]        | PWA-12.1.0 |
+| Bug   | Deployment Q | Devagouda Patil          | Venia Mega nav is broken in scaffolded app on develop branch.                         | [PWA-2286][] | [3513][]        | PWA-12.1.0 |
+| Task  | Deployment Q | Devagouda Patil          | Repo Metadata Service Onboarding: UPWARD-PHP                                          | [PWA-2251][] | [5][]           | PWA-12.1.0 |
+| Bug   | Deployment Q | Justin Conabree          | Cart remains active in browser memory on PWA site even after checkout                 | [PWA-2190][] | [3495][]        | PWA-12.1.0 |
+| Story | Deployment Q | Tommy Wiebell            | Parallelize Cypress tests on the CI                                                   | [PWA-2154][] | [3460][]        | PWA-12.1.0 |
+| Story | Deployment Q | James Calcaben           | Venia Sample Data metapackage deployVeniaSampleData.sh script update                  | [PWA-2151][] | [3473][]        | PWA-12.1.0 |
+| Task  | Deployment Q | Tommy Wiebell            | Add installation instructions to Venia Sample Data repo                               | [PWA-2141][] | [8][]           | PWA-12.1.0 |
+| Task  | Deployment Q | Bruce Denham             | Add local and Cloud installation instructions to meta packages                        | [PWA-2140][] | [4][], [1][]    | PWA-12.1.0 |
+| Bug   | Deployment Q |                          | Cart page out of stock crash                                                          | [PWA-2133][] | [3447][], [6][] | PWA-12.1.0 |
+| Story | Deployment Q |                          | Update Item Quantities                                                                | [PWA-2132][] | [3464][]        | PWA-12.1.0 |
+| Bug   | Deployment Q | Revanth kumar Annavarapu | Improve Venia "create" scaffold script to stop relying on NPM log output              | [PWA-2131][] | [3463][]        | PWA-12.1.0 |
+| Story | Deployment Q | Ievgen Kolesov           | Create Venia Sample Data metapackage                                                  | [PWA-2106][] | [2][]           | PWA-12.1.0 |
+| Story | Deployment Q | Tommy Wiebell            | GQL Support for Contact Us & Newsletter                                               | [PWA-2004][] | [5][]           | PWA-12.1.0 |
+| Story | Deployment Q | Justin Conabree          | Refactor Site Header to use Tailwind Theme                                            | [PWA-1885][] | [3472][]        | PWA-12.1.0 |
 
```

Comment:
This list needs to include https://github.com/magento/pwa-studio/pull/3535 ?

Reply:
Thanks, @dpatil-magento. Need to find out why my automation script didn't pick this one up...

Reply:
Ran the script again. Just wasn't there when I ran it last. :)

Reply:
@bdenham I merged this yesterday night that might be the reason :)